src/mmw/js/src/core/views.js

```diff
@@ -206,7 +216,8 @@ var MapView = Marionette.ItemView.extend({
         'change:size': 'toggleMapSize',
         'change:maskLayerApplied': 'toggleMask',
         'change:dataCatalogResults': 'renderDataCatalogResults',
-        'change:dataCatalogActiveResult': 'renderDataCatalogActiveResult',
+        'change:dataCatalogActiveResult': 'renderDataCatalogResult',
+        'change:dataCatalogDetailResult': 'renderDataCatalogResult',
```

Comment:
Since they both trigger the same function, consider putting them in one line:

```js
'change:dataCatalogActiveResult change:dataCatalogDetailResult': 'renderDataCatalogResult',
```

otherwise it seems like there is a mistake in the first one, where the left and right side don't match.

Reply:
Unfortunately, condensing them into one line modifies the shape of the event object. The changed attribute is no longer included in the event payload, which prevents the event handler from detecting which attribute is changing.

> otherwise it seems like there is a mistake in the first one, where the left and right side don't match.

I'm not sure I understand. Can you elaborate?